src/pages/chat/Chat.tsx

```diff
@@ -21,60 +21,74 @@ import ChatTextArea from '~/containers/chat/chat-text-area/ChatTextArea'
 
 import { defaultResponses } from '~/constants'
 import { styles } from '~/pages/chat/Chat.styles'
-import {
-  ChatResponse,
-  MessageInterface,
-  PositionEnum,
-  GetMessagesParams
-} from '~/types'
+import { ChatResponse, MessageInterface, PositionEnum } from '~/types'
 
 const Chat = () => {
   const { t } = useTranslation()
   const { isMobile } = useBreakpoints()
   const { openDrawer, closeDrawer, isOpen } = useDrawer()
   const [selectedChat, setSelectedChat] = useState<ChatResponse | null>(null)
-  const [textAreaValue, setTextAreaValueValue] = useState<string>('')
+  const [messages, setMessages] = useState<MessageInterface[] | []>([])
```

Comment:
you don't need to add `[]` in typing of this state, just leave `Message` interface

Reply:
also i think it would be better to rename interface of the message, just to `Message`

Reply:
leave only MessageInterface[], but I don`t think I need to rename it, because we use this Interface keyword in for many interfaces